app/_locales/en/messages.json

```diff
@@ -5687,6 +5687,12 @@
   "yourPrivateSeedPhrase": {
     "message": "Your Secret Recovery Phrase"
   },
+  "yourTransactionConfirmed": {
+    "message": "Your transaction is confirmed"
+  },
+  "yourTransactionJustConfirmed": {
+    "message": "Your transaction was just confirmed. We weren't able to cancel it."
```

Comment:
Could we get @coreyjanssen or @bschorchit to check the phrasing? "Was just confirmed" and "is confirmed" don't quite sound right to me.

Reply:
I will ping them in the slack thread once again, tbh I copy pasted the design

Reply:
How about this?

**Transaction already confirmed**

We weren't able to cancel your transaction before it was confirmed on the blockchain.

Reply:
Thanks @coreyjanssen! I fixed them all